utils/lobby.ts

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
maybe delete this since we are just using the spotify id as the lobby id

Reply:
please remove this